app/scripts/lib/offscreen-bridge/trezor-offscreen-bridge.ts

```diff
@@ -40,7 +42,8 @@ export class TrezorOffscreenBridge implements TrezorBridge {
         msg.target === OffscreenCommunicationTarget.extension &&
         msg.event === OffscreenCommunicationEvents.trezorDeviceConnect
       ) {
-        this.model = msg.payload;
+        this.model = msg.payload.model;
+        this.minorVersion = msg.payload.minorVersion;
```

Comment:
it looks like `msg.payload` was assigned to `this.model`, while now we are assigning `msg.payload.model`.

Would this still work with the standard Trezor device?

Reply:
yes, i have tested the Trezor device, and it still work, since i have already work on `trezor.ts` line 44 to change payload to contain model and minorVersion.